pom.xml

```diff
@@ -31,27 +31,48 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <java.version>1.8</java.version>
+
+    <java.version>8</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.version>3.6.3</maven.version>
-    <slf4j.version>1.7.16</slf4j.version>
-    <guava.version>14.0.1</guava.version>
-    <CodeCacheSize>512m</CodeCacheSize>
+
+    <checkstyle.version>8.23</checkstyle.version>
     <codahale.metrics.version>3.2.6</codahale.metrics.version>
+    <commons-lang3.version>3.10</commons-lang3.version>
+    <commons-io.version>2.8.0</commons-io.version>
+    <commons-crypto.version>1.0.0</commons-crypto.version>
+    <google.jsr305.version>1.3.9</google.jsr305.version>
+    <grpc.version>1.44.0</grpc.version>
+    <guava.version>14.0.1</guava.version>
     <javaxservlet.version>3.1.0</javaxservlet.version>
-    <!-- Apache Ratis version -->
+    <junit.version>4.12</junit.version>
+    <leveldb.version>1.8</leveldb.version>
+    <log4j.version>1.2.17</log4j.version>
+    <log4j2.version>2.17.1</log4j2.version>
+    <lz4-java.version>1.8.0</lz4-java.version>
+    <mockito.version>1.10.19</mockito.version>
```

Comment:
I found different and incompatible mockito versions are used across modules, will try to unify them in the following PRs